PyInstaller/hooks/hook-_tkinter.py

```diff
@@ -45,6 +45,59 @@ def _handle_broken_tk():
                 v['TIX_LIBRARY'] = abs_path
 
 
+def _handle_activetcl_install(tcl_root, tcltree):
```

Comment:
I'd prefer an different name, stating that this does only warn about activetcl-installs (e.g. `_warn_if_actvivetcl_or_teapot_install`). For me `_handle_activctcl_install` sounds like it would do things to package activetcl correctly.